source/plugins/languages/analyzers.mjs

```diff
@@ -75,9 +75,11 @@ export async function recent({login, data, imports, rest, account}, {skipped = [
         .flatMap(({payload}) => payload.commits)
         .filter(({author}) => data.shared["commits.authoring"].filter(authoring => author?.email?.toLocaleLowerCase().includes(authoring)||author?.name?.toLocaleLowerCase().includes(authoring)).length)
         .map(commit => commit.url)
-        .map(async commit => (await rest.request(commit)).data.files),
-    ),
+        .map(async commit => (await rest.request(commit)).data),
+    )
   ]
+  .filter(({parents}) => parents.length <= 1)
```

Comment:
Apparently the `parents` key is not always there:
![image](https://user-images.githubusercontent.com/42429413/132107381-22eea277-8cb0-4b0d-acce-c797048b49f5.png)